app/__init__.py

```diff
@@ -30,6 +36,27 @@ class Meta:
 mydb.connect()
 mydb.create_tables([TimelinePost])
 
+@app.route('/api/timeline_post', methods=['POST'])
+def post_time_post():
+    name = request.form['name']
+    email = request.form['email']
+    content = request.form['content']
```

Comment:
@cjohnson74 i noticed validations are missing. There are no handling of malformed timeline post. could you take a look?